src/amber/server/server.cr

```diff
@@ -0,0 +1,102 @@
+module Amber
+  class Server
+    include Amber::DSL::Server
+    alias WebSocketAdapter = WebSockets::Adapters::RedisAdapter.class | WebSockets::Adapters::MemoryAdapter.class
+    property pubsub_adapter : WebSocketAdapter = WebSockets::Adapters::MemoryAdapter
+    property settings : Amber::Settings
+    getter handler = Pipe::Pipeline.new
+    getter router = Router::Router.new
+
+    def self.instance
+      @@instance ||= new(Amber.settings)
+    end
+
+    def self.start
+      instance.run
+    end
+
+    # Configure should probably be deprecated in favor of settings.
+    def self.configure
+      with self yield settings
+    end
+
+    def self.settings
```

Comment:
So this is all back here now? I'm fine with it. Why did you move it out to `configuration.cr` just to move it back here though?

Reply:
Removing this in favor of `Amber.settings`